generators/app/templates/imports/lib/config/layout.js

```diff
@@ -0,0 +1,21 @@
+BlazeLayout.setRoot("body");
+
+Meteor.startup(function () {
+  const bodyClass = _bodyClass(FlowRouter.getRouteName());
+
+  Blaze.addBodyClass(bodyClass);
+
+  FlowRouter.triggers.enter([function (context) {
+    Blaze.addBodyClass(_bodyClass(context.route.name));
+  }]);
+
+  FlowRouter.triggers.exit([function (context) {
+    $("body > header").css({
+      top: 0
+    });
+  }]);
```

Comment:
can you run `standard --fix` on this file? By `standard`'s standard, we use single quotes and no semicolons.

See: [Standard](https://www.npmjs.com/package/standard)